app/icons/Add.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { forwardRef } from 'react'
+import type { SVGProps } from 'react'
+
+export default forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(function Bed(props, ref) {
+  return (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor"
+         strokeWidth="2" strokeLinecap="round" strokeLinejoin="round" ref={ref} {...props}>
```

Comment:
Similar formatting as described above, for icons this isn't really all that important though